rdtools/analysis_chains.py

```diff
@@ -717,6 +744,8 @@ def _clearsky_preprocess(self):
         except AttributeError:
             raise AttributeError("No poa_global_clearsky. 'set_clearsky' must be run " +
                                  "prior to 'clearsky_analysis'")
+        if 'csi_filter' not in self.filter_params:
+            self.filter_params['csi_filter'] = {}
```

Comment:
This is another results-changing difference I'm not sure we should enable without a major version increment.

Reply:
Another question: isn't it valid to do a "clearsky-based" analysis without necessarily using the clear-sky filter?

Reply:
This is removed - it's no longer logically necessary since we use a different key sensor_csi_filter.  so csi_filter can be enabled by default, but removed by the user in clearsky analysis if they really wanted to for some reason.